tests/test_unit_conversion.py

```diff
@@ -15,5 +15,8 @@ def test_add_preferred_custom_unit_name(model):
     assert str(model.units.summarise_units(time_var)) == "ms"
     model.units.add_preferred_custom_unit_name('millisecond', [{'prefix': 'milli', 'units': 'second'}])
     assert str(model.units.summarise_units(time_var)) == "millisecond"
+    ''' add_custom_unit does not allow adding already existing units but add_preferred_custom_unit_name does since we
```

Comment:
Using `# comment` style is probably more appropriate than docstring style here?

Reply:
is it? from what i read unless they're on top they don't get treated as docstring